app/models/pipeline_run.rb

```diff
@@ -578,6 +578,12 @@ def update_job_status
     save
   end
 
+  def job_status_display
+    return "Pipeline Initializizng" unless self.job_status
+    stage = self.job_status.to_s.split("-")[0].split(".")[1]
```

Comment:
actually job_status could have like "4.De-Novo Assembly-FAILED|READY" or "CHECKED" and stuff right?

Reply:
yes. so if it's CHECKED, it will return nil for stage. it will just return the status.